Server/crafting.txt

```diff
@@ -250,7 +257,7 @@ GoldenChestplate     = GoldIngot, 1:1, 3:1, 1:2, 2:2, 3:2, 1:3, 2:3, 3:3
 IronChestplate       = IronIngot, 1:1, 3:1, 1:2, 2:2, 3:2, 1:3, 2:3, 3:3
 LeatherChestplate    = Leather,   1:1, 3:1, 1:2, 2:2, 3:2, 1:3, 2:3, 3:3
 
-# Leggins:
```

Comment:
Well, it should really be _leggings_.

I changed it because all of the items below had pants in their name and none had leggings.